near/omni-tests/src/lib.rs

```diff
@@ -219,9 +219,7 @@ mod tests {
         let required_deposit_for_fin_transfer = NEP141_DEPOSIT
             .saturating_mul(storage_deposit_accounts.len() as u128)
             .saturating_add(required_balance_for_fin_transfer)
-            .saturating_add(NearToken::from_yoctonear(2))
-            // This is a temporary value to cover the issue with incorrect storage deposit calculation
-            .saturating_add(NearToken::from_yoctonear(1250000000000000000000));
+            .saturating_add(NearToken::from_yoctonear(2));
```

Comment:
Let's improve the UX and add these two yocto to the `pub fn required_balance_for_fin_transfer(&self) -> NearToken`

Reply:
Thanks to @olga24912 this change was made in #138. I've updated the PR to remove the excess storage deposit.